CMakeLists.txt

```diff
@@ -1002,7 +1143,18 @@ if(WOLFSSL_BASE64_ENCODE)
     list(APPEND WOLFSSL_DEFINITIONS "-DWOLFSSL_BASE64_ENCODE")
 endif()
 
-# TODO: - Base16
+# Base16
+add_option("WOLFSSL_BASE16_ENCODE"
```

Comment:
Change WOLFSSL_BASE16_ENCODE to WOLFSSL_BASE16. Just matches up better with configure.ac.